tools/devops/automation/scripts/GitHub.psm1

```diff
@@ -442,16 +445,21 @@ function New-GitHubSummaryComment {
         $sb.AppendLine("* [Html Report (VSDrops)]($Env:VSDROPS_INDEX) [Download]($Env:SYSTEM_TEAMFOUNDATIONCOLLECTIONURI$Env:SYSTEM_TEAMPROJECT/_apis/build/builds/$Env:BUILD_BUILDID/artifacts?artifactName=HtmlReport-sim&api-version=6.0&`$format=zip)")
     }
     if (-not [string]::IsNullOrEmpty($APIDiff)) {
-        WriteDiffs $sb $APIDiff
+        WriteDiffs $sb "API diff" $APIDiff
     } else {
         Write-Host "API diff urls have not been provided."
     }
+    if (-not [string]::IsNullOrEmpty($APIGeneratorDiffJson)) {
+        WriteDiffs $sb "API Current PR diff" $APIGeneratorDiffJson
+    } else {
+        Write-Host "API Current PR diff urls have not been provided."
+    }
```

Comment:
Why check call -not, there is not need, invert the code.

Reply:
You mean as in:
```
   if ([string]::IsNullOrEmpty($APIGeneratorDiffJson)) {
        Write-Host "API Current PR diff urls have not been provided."
    } else {
        WriteDiffs $sb "API Current PR diff" $APIGeneratorDiffJson
    }
 ```